src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/apps/AssociationTypeApp.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {DependenciesProvider} from '@akeneo-pim-community/legacy-bridge';
+import {ThemeProvider} from 'styled-components';
+import {pimTheme} from 'akeneo-design-system';
+import {AssociationTypesIndex} from '@akeneo-pim-community/settings-ui';
+
+const AssociationTypeApp = () => {
```

Comment:
In order to be consistent, could we keep plurial?

```suggestion
const AssociationTypesApp = () => {
```